src/components/Toaster/index.js

```diff
@@ -5,11 +5,9 @@ import {
   ToastContainer as BaseToastContainer,
   toast as baseToast,
 } from 'react-toastify'
-import { injectStyle } from 'react-toastify/dist/inject-style'
 import Icon from '../Icon'
 import NotificationBar from '../NotificationBar'
-
-injectStyle()
+import 'react-toastify/dist/ReactToastify.css'
```

Comment:
I got a doubt on this change, i think we use to be like this before but it's doesn't work properly on other projects